run_dir/static/js/flowcell_samples.js

```diff
@@ -43,7 +32,18 @@ $.getJSON("/api/v1/flowcell_info2/"+flowcell, function(data) {
 
     // If demultiplexed, get additional information for each lane
     if(data.hasOwnProperty("demuldone")){
-
+        // tresholds for the number of clusters: depending on the run mode
+        var run_mode = data['run_mode'];
+        var treshold = 0;
+        if (run_mode == 'HiSeq X') {
+            treshold = 320;
+        } else if (run_mode == 'RapidHighOutput') {
+            treshold = 188;
+        } else if (run_mode == 'RapidRun') {
+            treshold = 114;
+        } else if (run_mode == 'HighOutput') {
+            treshold = 143;
+        }
```

Comment:
These tresholds are taken from the doc 1244. Should be correct, but maybe Francesco can check it once again, @vezzi ?